README.md

```diff
@@ -50,7 +50,9 @@ ignore list will be disabled and turned off on a system. The following are
 exceptions:
 
 ```
-    The following services are hard coded to never be killed by svckill:
+    The following services are hard coded to never be killed by svckill
```

Comment:
I think this list should be removed and a more clear description of how to access the list (data in modules) would be best.